package.json

```diff
@@ -49,7 +49,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 8.11",
+    "node": ">= 10.0",
```

Comment:
Put this back to `8.11` if you want to test this as part of CI - this is related to the Node version you have installed locally, which is independent of the Node version embedded in Electron.